### PR TITLE
Update dependency versions in samples monthly

### PR DESCRIPTION
**Pull request by backwind1233**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,70 +13,70 @@
...
-| AAD              | [azure-spring-boot-starter-active-directory:3.7.0]                                             | [aad-web-application](aad/azure-spring-boot-starter-active-directory/aad-web-application)                                         |
-| AAD              | [azure-spring-boot-starter-active-directory:3.7.0]                                             | [aad-webapp-resource-server](aad/azure-spring-boot-starter-active-directory/aad-web-application-and-resource-server)              |
-| appconfiguration | [azure-spring-boot-starter-cosmos:3.7.0]                                                       | [azure-appconfiguration-conversion-sample-initial](appconfiguration/azure-appconfiguration-conversion-sample-initial)             |
+| AAD              | [azure-spring-boot-starter-active-directory-b2c:3.8.0]                                         | [aad-b2c-resource-server](aad/azure-spring-boot-starter-active-directory-b2c/aad-b2c-resource-server)                             |
```
Comment (chenrujun) [inline README.md:16]: Did you check these versions manually one by one?

File: README.md [hunk 2]
```diff
@@ -13,70 +13,70 @@
...
+| AAD              | [azure-spring-boot-starter-active-directory:3.8.0]                                             | [aad-resource-server](aad/azure-spring-boot-starter-active-directory/aad-resource-server)                                         |
+| AAD              | [azure-spring-boot-starter-active-directory:3.8.0]                                             | [aad-web-application](aad/azure-spring-boot-starter-active-directory/aad-web-application)                                         |
+| AAD              | [azure-spring-boot-starter-active-directory:3.8.0]                                             | [aad-webapp-resource-server](aad/azure-spring-boot-starter-active-directory/aad-web-application-and-resource-server)              |
+| appconfiguration | [azure-spring-boot-starter-cosmos:3.8.0]                                                       | [azure-appconfiguration-conversion-sample-initial](appconfiguration/azure-appconfiguration-conversion-sample-initial)             |
```
Comment (chenrujun) [inline README.md:24]: Is this a mistake?

File: aad/azure-spring-boot-starter-active-directory/aad-resource-server-by-filter/pom.xml [hunk 3]
```diff
@@ -23,7 +23,7 @@
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core</artifactId>
-        <version>1.18.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+        <version>1.19.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
```
Comment (chenrujun) [inline aad/azure-spring-boot-starter-active-directory/aad-resource-server-by-filter/pom.xml:26]: These items are updated by script, right?

File: aad/azure-spring-boot-starter-active-directory/aad-resource-server/pom.xml [hunk 4]
```diff
@@ -19,7 +19,7 @@
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core</artifactId>
-        <version>1.18.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+        <version>1.19.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
```
Comment (chenrujun) [inline aad/azure-spring-boot-starter-active-directory/aad-resource-server/pom.xml:22]: How about import bom instead of write these versions?

**Review by chenrujun — CHANGES_REQUESTED** [review]

File: aad/azure-spring-boot-starter-active-directory/aad-resource-server-by-filter/pom.xml (hunk 3 above)
Comment (backwind1233) [reply aad/azure-spring-boot-starter-active-directory/aad-resource-server-by-filter/pom.xml:26]: yes

File: aad/azure-spring-boot-starter-active-directory/aad-resource-server/pom.xml (hunk 4 above)
Comment (backwind1233) [reply aad/azure-spring-boot-starter-active-directory/aad-resource-server/pom.xml:22]: I will do it this month

File: README.md (hunk 1 above)
Comment (backwind1233) [reply README.md:16]: yes

File: README.md (hunk 2 above)
Comment (backwind1233) [reply README.md:24]: actually,azure-appconfiguration-conversion-sample-initial depends on azure-spring-boot-starter-cosmos
image

Comment (chenrujun) [reply README.md:24]: The folder/artifact name is confusing.

**Review by chenrujun — APPROVED** [review]

**Pull request merged**